library/xml-model/src/main/java/org/cru/godtools/xml/model/Manifest.kt

```diff
@@ -137,31 +139,44 @@ class Manifest : BaseModel, Styles {
 
         // process any child elements
         var title: Text? = null
-        var categoriesData: List<Category>? = null
-        var pagesData: PagesData? = null
-        var resourcesData: Map<String?, Resource>? = null
-        var tipsData: List<Tip>? = null
-        while (parser.next() != XmlPullParser.END_TAG) {
-            if (parser.eventType != XmlPullParser.START_TAG) continue
-
-            when (parser.namespace) {
-                XMLNS_MANIFEST -> when (parser.name) {
-                    XML_TITLE -> title = Text.fromNestedXml(this, parser, XMLNS_MANIFEST, XML_TITLE)
-                    XML_CATEGORIES -> categoriesData = parseCategories(parser)
-                    XML_PAGES -> pagesData = parsePages(parser, parseFile)
-                    XML_RESOURCES -> resourcesData = parseResources(parser)
-                    XML_TIPS -> tipsData = parser.parseTips(parseFile)
+        val aemImports = mutableListOf<Uri>()
+        val categories = mutableListOf<Category>()
```

Comment:
What's the value of changing all of these to ```mutableListOf``` and not setting a default value of null?

Reply:
only difference is if we get malformed XML that has multiple `<categories>` or `<pages>` nodes, this new logic will aggregate all the nodes together, the old way would just return the most recently parsed one.

I think I changed it to stay consistent with how I implemented the `tipsTasks` and `tractPageTasks` intermediary variables.